code/game/objects/random/random.dm

```diff
@@ -278,3 +276,234 @@
+					/obj/item/clothing/under/color/green,\
+					/obj/item/clothing/under/color/grey,\
+					/obj/item/clothing/under/color/orange,\
+					/obj/item/clothing/under/color/pink,\
+					/obj/item/clothing/under/color/red,\
+					/obj/item/clothing/under/color/white,\
+					/obj/item/clothing/under/color/yellow,\
+					/obj/item/clothing/under/lightblue,\
+					/obj/item/clothing/under/aqua,\
+					/obj/item/clothing/under/purple,\
+					/obj/item/clothing/under/lightpurple,\
+					/obj/item/clothing/under/lightgreen,\
+					/obj/item/clothing/under/lightbrown,\
+					/obj/item/clothing/under/brown,\
+					/obj/item/clothing/under/yellowgreen,\
+					/obj/item/clothing/under/darkblue,\
+					/obj/item/clothing/under/lightred,\
+					/obj/item/clothing/under/darkred)
+
+/obj/random/loot/item_to_spawn()
+	name = "Random Maintenance Loot Items"
+	desc = "Stuff for the maint-dwellers."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "gift3"
+	var/list/maint = list("/obj/item/clothing/glasses/meson" = 1,
+	"/obj/item/clothing/glasses/meson/prescription" = 0.7,
+	"/obj/item/clothing/glasses/material" = 0.8,
+	"/obj/item/clothing/glasses/sunglasses" = 1.5,
+	"/obj/item/clothing/glasses/welding" = 1.2,
+	"/obj/item/clothing/under/captain_fly" = 0.7,
+	"/obj/item/clothing/under/rank/mailman" = 0.6,
+	"/obj/item/clothing/under/rank/vice" = 0.8,
+	"/obj/item/clothing/under/assistantformal" = 1,
+	"/obj/item/clothing/under/rainbow" = 0.9,
+	"/obj/item/clothing/under/overalls" = 1,
+	"/obj/item/clothing/under/redcoat" = 0.5,
+	"/obj/item/clothing/under/serviceoveralls" = 1,
+	"/obj/item/clothing/under/psyche" = 0.5,
+	"/obj/item/clothing/under/track" = 0.9,
+	"/obj/item/clothing/under/rank/dispatch" = 1,
+	"/obj/item/clothing/under/syndicate/tacticool" = 1,
+	"/obj/item/clothing/under/syndicate/tracksuit" = 0.2,
+	"/obj/item/clothing/accessory/badge" = 0.2,
+	"/obj/item/clothing/accessory/badge/old" = 0.2,
+	"/obj/item/clothing/accessory/storage/webbing" = 0.6,
+	"/obj/item/clothing/accessory/storage/knifeharness" = 0.3,
+	"/obj/item/clothing/head/collectable/petehat" = 0.3,
+	"/obj/item/clothing/head/hardhat" = 1.5,
+	"/obj/item/clothing/head/redcoat" = 0.4,
+	"/obj/item/clothing/head/syndicatefake" = 0.5,
+	"/obj/item/clothing/head/richard," = 0.3,
+	"/obj/item/clothing/head/soft/rainbow" = 0.7,
+	"/obj/item/clothing/head/plaguedoctorhat" = 0.5,
+	"/obj/item/clothing/head/cueball" = 0.5,
+	"/obj/item/clothing/head/pirate" = 0.4,
+	"/obj/item/clothing/head/bearpelt" = 0.4,
+	"/obj/item/clothing/head/witchwig" = 0.5,
+	"/obj/item/clothing/head/pumpkinhead" = 0.6,
+	"/obj/item/clothing/head/kitty" = 0.2,
+	"/obj/item/clothing/head/ushanka" = 0.6,
+	"/obj/item/clothing/head/helmet/augment" = 0.1,
+	"/obj/item/clothing/mask/balaclava" = 1,
+	"/obj/item/clothing/mask/gas" = 1.5,
+	"/obj/item/clothing/mask/gas/cyborg" = 0.7,
+	"/obj/item/clothing/mask/gas/owl_mask" = 0.8,
+	"/obj/item/clothing/mask/gas/syndicate" = 0.4,
+	"/obj/item/clothing/mask/fakemoustache" = 1,
```

Comment:
Doesn't the fake moustache allow you to change your name and stuff? Ergo, is a syndicate item?